tools/selproj/client.go

```diff
@@ -0,0 +1,77 @@
+//go:build tools
+
+package main
+
+import (
+	"context"
+
+	"github.com/aiven/aiven-go-client/v2"
+)
+
+// AivenClient defines the interface for Aiven client operations.
+// It is implemented by AivenClientAdapter.
+type AivenClient interface {
+	Projects() ProjectsHandler
+	Services() ServicesHandler
+}
```

Comment:
Let's add vpc if we now have such a client.

Reply:
Will do if it fails somehow in future.